### PR TITLE
[USH-1511] Use only parameters for posts in Mobile

### PR DESCRIPTION
**Pull request by Ifycode**

### CODE REVIEW

**Comment by Ifycode** [comment]
Marking as ready for review, since it's just to merge in the web ticket for the shared variables/interfaces to make it to work @Angamanga

**Review by Angamanga — COMMENTED** [review]
I had to merge USH-1510 into this pr as we discussed in our call, after that all works fine! 
Hold off merging this until the PR for USH-1510 is approved and development is merged into mobile-development 🙌

**Review by Angamanga — APPROVED** [review]
Works fine with the updates from the development-branch 🙌

**Pull request merged**
